packages/util/src/get-pingback-id.ts

```diff
@@ -0,0 +1,22 @@
+import { v4 as uuid } from 'uuid'
+let pingbackId = ''
+const getPingbackId = () => {
+    // it exists in memory
+    if (!pingbackId) {
+        try {
+            // it exists in storage
+            pingbackId = sessionStorage.getItem('giphyPingbackId') as string
+        } catch (_) {}
+        if (!pingbackId) {
+            // we need to create it
+            pingbackId = uuid().replace(/-/g, '').substring(0, 16)
```

Comment:
keep the uuid the required length 

@seanpquig 16 length doesn't ensure uniqueness 100%, but assuming that's okay

Reply:
ooooo

Reply:
might need to add time badck into the mix

Reply:
@cosmocochrane1 @seanpquig if it needs to be 16 digits, we can't use uuid, but we can just generate a random number using a lib, something from here: https://www.npmjs.com/search?q=random+number 

lmk if you want to use a random number that is 16 digits and I'll make it happen with the first option in npm called `brorand`